addons/spreadsheet/static/tests/pivots/model/pivot_plugin_test.js

```diff
@@ -1193,4 +1193,191 @@ QUnit.module("spreadsheet > pivot plugin", {}, () => {
+        assert.strictEqual(getEvaluatedCell(model, "A1").format, "#,##0.00");
+        model.dispatch("UPDATE_PIVOT", {
+            pivotId,
+            pivot: {
+                ...model.getters.getPivotDefinition(pivotId),
+                measures: [{ name: "probability", aggregator: "count_distinct" }],
+            },
+        });
+        await nextTick();
+        assert.strictEqual(getEvaluatedCell(model, "A1").format, "0");
+    });
+
+    QUnit.test("changing order of group by", async (assert) => {
+        const { model, pivotId } = await createSpreadsheetWithPivot({
+            arch: /* xml */ `
+                <pivot>
+                    <field name="foo" type="col"/>
+                    <field name="probability" type="measure"/>
+                </pivot>`,
+            mockRPC: async function (route, args) {
+                if (args.method === "read_group") {
+                    assert.step(args.kwargs.orderby || "NO_ORDER");
+                }
+            },
+        });
+        assert.verifySteps(["NO_ORDER", "NO_ORDER"]);
+        model.dispatch("UPDATE_PIVOT", {
+            pivotId,
+            pivot: {
+                ...model.getters.getPivotDefinition(pivotId),
+                columns: [{ name: "foo", order: "asc" }],
+            },
+        });
+        assert.deepEqual(model.getters.getPivotDefinition(pivotId).columns, [
+            { name: "foo", order: "asc" },
+        ]);
+        await nextTick();
+        assert.verifySteps(["NO_ORDER", "foo asc"]);
+        model.dispatch("UPDATE_PIVOT", {
+            pivotId,
+            pivot: {
+                ...model.getters.getPivotDefinition(pivotId),
+                columns: [{ name: "foo" }],
+            },
+        });
+        await nextTick();
+        assert.verifySteps(["NO_ORDER", "NO_ORDER"]);
+    });
+
+    QUnit.test("change date order", async (assert) => {
+        const { model, pivotId } = await createSpreadsheetWithPivot({
+            arch: /* xml */ `
+                <pivot>
+                    <field name="probability" type="measure"/>
+                </pivot>`,
+            mockRPC: async function (route, args) {
+                if (args.method === "read_group") {
+                    assert.step(args.kwargs.orderby || "NO_ORDER");
+                }
+            },
+        });
+        assert.verifySteps(["NO_ORDER"]);
+        model.dispatch("UPDATE_PIVOT", {
+            pivotId,
+            pivot: {
+                ...model.getters.getPivotDefinition(pivotId),
+                columns: [
+                    { name: "date", granularity: "year", order: "asc" },
+                    { name: "date", granularity: "month", order: "desc" },
+                ],
+            },
+        });
+        await nextTick();
+        assert.verifySteps(["NO_ORDER", "date:year asc", "date:year asc,date:month desc"]);
+    });
+
+    QUnit.test("duplicated dimension on col and row with different granularity", async (assert) => {
```

Comment:
do we have a test with multiple measures on the same field with different aggregates ?